reference/features/search_parameters.md

```diff
@@ -42,6 +42,18 @@ When `q` isn't specified, a **placeholder search** is performed. Placeholder sea
 
 Placeholder search returns the documents that best match other search parameters, sorted according to that index's [ranking rules](/reference/features/settings.md#ranking-rules). This feature also supports [faceting](/reference/features/faceted_search.md) and [filtering](/reference/features/filtering.md)
 
+### Phrase search
+
+If you enclose search terms in double quotes (`"`), MeiliSearch will only return documents containing those terms in the order they were given. This is called a **phrase search**.
+
+Phrase searches are case-insensitive and ignore [soft separators such as `-`, `,`, and `:`](/reference/under_the_hood/datatypes.md). Using a hard separator within a phrase search effectively splits it into multiple separate phrase searches: `"Octavia.Butler"` will return the same results as `"Octavia" "Butler"`.
+
+You can combine phrase search and normal queries in a single search request. In this case, MeiliSearch will first fetch all documents with exact matches to the given phrase(s), and [then proceed with its default behavior](/learn/core_concepts/relevancy.md).
```

Comment:
I'm having second thoughts about this last sentence, not sure it adds anything. What do you think of getting rid of it?

```suggestion
You can combine phrase search and normal queries in a single search request.
```

Reply:
I'm leaning towards leaving it because this explains exactly why the example (`"african american" poet`) works.

Reply:
Hmm, I don't completely understand but it's probably just the sleep deprivation talking.

Reply:
`"African American" poet` works well because it matches `African American` strictly , but uses the regular MeiliSearch process for `poet`, meaning it will match also match words like `poem` (possibly also `poetic` and `poetry`? Not 100% sure on those two, to be honest).

Reply:
Hey @guimachiavelli! In your example `poet` has less than 5 characters, meaning that we don't allow a typo on this word 😅 but if the word would be longer we would allow 1 typo on this word.

Meanwhile, because `poet` is not in the `".."`, meilisearch will return documents where `poet` isn't near `African American`.

Reply:
@ManyTheFish I stand corrected! Thanks for that!